spec/policies/claims/claim_policy_spec.rb

```diff
@@ -101,6 +101,32 @@
     end
   end
 
+  permissions :change? do
+    context "when user has an internal draft claim" do
+      it "grants access" do
+        expect(claim_policy).not_to permit(support_user, internal_draft_claim)
+      end
+    end
+
+    context "when user has a draft claim" do
+      it "grants access" do
+        expect(claim_policy).to permit(support_user, draft_claim)
+      end
+    end
+
+    context "when user has a submitted claim" do
+      it "denies access" do
+        expect(claim_policy).not_to permit(support_user, submitted_claim)
+      end
+    end
+
+    context "when user is not a support user" do
+      it "denies access" do
+        expect(claim_policy).not_to permit(user, submitted_claim)
```

Comment:
If we're checking a `submitted_claim`, it will _always_ be denied regardless of the type of user.

```suggestion
        expect(claim_policy).not_to permit(user, draft_claim)
```